apps/etterlatte-saksbehandling-ui/client/src/components/person/KlageListe.tsx

```diff
@@ -65,6 +65,8 @@ function KlageTabell(props: { klager: Array<Klage> }) {
     return null
   }
 
+  klager.sort((a, b) => (new Date(a.opprettet) < new Date(b.opprettet) ? 1 : -1))
```

Comment:
Tror det er best at denne komponenten ikke modifiserer props den tar inn, så ville foretrukket å bruke `toSorted` i stedet for, eventuelt sortere klagene i bruken av komponenten.